spec/helpers/step_nav_actions_helper_spec.rb

```diff
@@ -18,5 +20,25 @@
       step_by_step_page.review_requester_id = user.uid
       expect(helper.can_review?(step_by_step_page, user)).to be false
     end
+
+    it "returns true if another user tries to claim the review" do
+      step_by_step_page.status = "submitted_for_2i"
+      step_by_step_page.review_requester_id = user.uid
```

Comment:
Could you move this line into the `before` block as it's used in all the scenarios?